### ISSUE TITLE
Map not loaded completely at some zoom levels

### ISSUE THREAD

**Issue by andrcoding** [cgeo/cgeo#14867]
### Describe your problem!

Using the map I sometimes get some visual issues like this here:
![image](https://github.com/cgeo/cgeo/assets/78544594/2964d142-238d-4893-ab45-1f51c6aca79d)
This seems to be device-independent as it occurs on two different devices

### How to reproduce?

In case the visual issue happens it is always at the same zoom level. However, I cannot reproduce this effect on purpose.

### Actual result after these steps?

-

### Expected result after these steps?

no grey pattern

### Reproducible

No

### c:geo Version

2023.11.01

**Comment by devemux86** [comment]
Could be related to https://github.com/cgeo/cgeo/issues/13593

**Comment by andrcoding** [comment]
Yes, seems to be very similar. Except I could not reproduce it and it just happened since this autumn. 
I close this here as the other one is still open

**Comment by fm-sys** [comment]
Wait. From your screenshot I can see that you are using our "official" map implementation, while  #13593 should only happen for on VTM which is part of a new (internally called unified) map implementation not yet released.

@andrcoding could you tell us which map (and theme?) you are using?
And just to be sure, you have not intentionally opted in to use our new map implementation via a hidden preference?

**Comment by andrcoding** [comment]
I use the "Deu+west Freizeitkarte" downloaded directly via cgeo. The design is the "default freizeitkarte" theme.
No, I haven't changed anything in a hidden menu. It is my setting from 2 or 3 years ago (except the offline map which I have updated in September/October).

**Comment by moving-bits** [comment]
Won't fix. Old map implementations are deprecated as of release 2025.07.17 and superseded by UnifiedMap.
